src/ResourceManager/DevSpaces/Commands.DevSpaces/Generated/ControllersOperations.cs

```diff
@@ -0,0 +1,1953 @@
+// <auto-generated>
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.DevSpaces
```

Comment:
Why are you generating an SDK here? Do you have a published .NET SDK?

Reply:
using Nuget package now.